test/connector/connector/uniswap/test_uniswap_connector.py

```diff
@@ -60,7 +60,7 @@ def setUpClass(cls):
         cls.clock: Clock = Clock(ClockMode.REALTIME)
         cls.connector: UniswapConnector = UniswapConnector(
             [trading_pair],
-            "0xdc393a78a366ac53ffbd5283e71785fd2097807fef1bc5b73b8ec84da47fb8de",
+            "0xdc393a78a366ac53ffbd5283e71785fd2097807fef1bc5b73b8ec84da47fb8de",  # noqa: mock
```

Comment:
This is an actual private key and not all of the test cases are mocks. Shouldn't this be marked with a TODO and a related task item instead?